cmd/kyma/install/cmd.go

```diff
@@ -1067,7 +1067,15 @@ func (cmd *command) patchMinikubeIP(minikubeIP string) error {
 }
 
 func (cmd *command) createOwnDomainConfigMap() error {
-	_, err := cmd.K8s.Static().CoreV1().ConfigMaps("kyma-installer").Create(&corev1.ConfigMap{
+	cm, err := cmd.K8s.Static().CoreV1().ConfigMaps("kyma-installer").Get("owndomain-overrides", metav1.GetOptions{})
+	if err == nil && cm != nil {
+		fmt.Println("ConfigMap already exists")
```

Comment:
Would it be necessary to update the config map in this case? Or can we just reuse the existing one?

Reply:
This ConfigMap reflects the information about the cluster it is deployed on. Since the cluster has not changed, it does not need to be updated.

Reply:
In case the user wants to install kyma with another domain, it makes sense to update this configmap.